pkg/components/imguploader/imguploader.go

```diff
@@ -73,8 +73,13 @@ func NewImageUploader() (ImageUploader, error) {
 
 		keyFile := gcssec.Key("key_file").MustString("")
 		bucketName := gcssec.Key("bucket").MustString("")
+		path := gcssec.Key("path").MustString("")
 
-		return NewGCSUploader(keyFile, bucketName), nil
+		if path != "" && path[len(path)-1:] != "/" {
```

Comment:
This can be removed if you use `path.Join` in the uploader. It will take care of inserting `/` if needed.

Reply:
Ok, Thank you.